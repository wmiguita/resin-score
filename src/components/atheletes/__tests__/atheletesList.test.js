import React from "react"
import renderer from "react-test-renderer"

import AtheletesList from "../atheletesList"

describe( "AtheletesList", () => {

  it( "should render AtheleteList", () => {
    const tree = renderer.create( <AtheletesList list={ [] } /> ).toJSON()

    expect( tree ).toMatchSnapshot()
  })
})
