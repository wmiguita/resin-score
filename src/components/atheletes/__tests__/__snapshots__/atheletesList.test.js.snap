// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtheletesList should render AtheleteList 1`] = `
<ul
  className="MuiList-root MuiList-padding"
/>
`;
